src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -58,9 +60,17 @@ def compress(self):
         The model will be instrumented and user should never edit it after calling this method.
         `self.modules_to_compress` records all the to-be-compressed layers
         """
+        if self.modules_wrapper is not None:
+            # already compressed
+            return
+        else:
+            self.modules_wrapper = []
+
         modules_to_compress = self.detect_modules_to_compress()
         for layer, config in modules_to_compress:
-            self._instrument_layer(layer, config)
+            wrapper = self._instrument_layer(layer, config)
```

Comment:
`_instrument_layer` is no longer a proper name

Reply:
I rename it to `_wrap_modules`, which seems more reasonable.